### PR TITLE
Append cli command to export domaines and identities to csv

### PR DESCRIPTION
**Pull request by harobed**
Append cli command to export domaines and identities to csv.

### CODE REVIEW

File: modoboa_admin/management/commands/export_domains.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import sys
+import csv
+from optparse import make_option
+
+from django.core.management.base import BaseCommand
+
+
```
Comment (tonioo) [inline modoboa_admin/management/commands/export_domains.py]: One empty line is enough :)

File: modoboa_admin/management/commands/export_identities.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import sys
+import csv
+from optparse import make_option
+
+from django.core.management.base import BaseCommand
+
+
```
Comment (tonioo) [inline modoboa_admin/management/commands/export_identities.py]: Same

File: modoboa_admin/management/commands/export_domains.py (hunk 1 above)
Comment (harobed) [inline modoboa_admin/management/commands/export_domains.py]: I use flake8 syntax checker. If I remove this line, it's a mistake.

It's 2 lines before class definition.

Comment (tonioo) [inline modoboa_admin/management/commands/export_domains.py]: But the line just after is another import statement...

Comment (harobed) [inline modoboa_admin/management/commands/export_domains.py]: I speak about lines 6 and 7

**Comment by harobed** [comment]
Coding style fixed.

**Comment by harobed** [comment]
Thanks

**Pull request merged**